### ISSUE TITLE
compass-import-once causes error when recompiling second stylesheet to reference same sprite

### ISSUE THREAD

**Issue by altearius** [Compass/compass#1720]
I am encountering an error when using `compass-import-once`.  The error occurs when there are two separate, independent stylesheets that both `@import` the same sprite file.

A clean compilation succeeds.  The second compilation always fails with this message:

```
Errno::ENOENT on line ["25"] of C: No such file or directory - small/*.png
```

Here is a complete log with `--trace` turned on:

```
C:\Temp\compass\importOnce>compass version
Compass 0.12.6 (Alnilam)
Copyright (c) 2008-2014 Chris Eppstein
Released under the MIT License.
Compass is charityware.
Please make a tax deductable donation for a worthy cause: http://umdf.org/compass

C:\Temp\compass\importOnce>compass clean && compass compile && echo -------- && compass compile --trace
   remove .sass-cache/
   remove stylesheets/Admin.css
   remove stylesheets/Public.css
   remove images/small-sc164b1b6fc.png
   create images/small-sc164b1b6fc.png
   create stylesheets/Admin.css
unchanged images/small-sc164b1b6fc.png
   create stylesheets/Public.css
--------
unchanged sass/Admin.scss
Errno::ENOENT on line ["25"] of C: No such file or directory - small/*.png
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-import-once-1.0.4/lib/compass/import-once/importer.rb:25:in `mtime'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:134:in `mtime'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:163:in `block in dependency_updated?'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:125:in `each'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:125:in `any?'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:125:in `dependencies_stale?'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:166:in `block in dependency_updated?'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:81:in `call'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:81:in `stylesheet_modified_since?'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/sass-3.2.19/lib/sass/plugin/staleness_checker.rb:69:in `stylesheet_needs_update?'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/compiler.rb:67:in `needs_update?'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/compiler.rb:148:in `should_compile?'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/compiler.rb:117:in `compile_if_required'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/compiler.rb:103:in `block (2 levels) in run'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/compiler.rb:101:in `each'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/compiler.rb:101:in `block in run'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/compiler.rb:126:in `timed'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/compiler.rb:100:in `run'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/commands/update_project.rb:45:in `perform'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/commands/base.rb:18:in `execute'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/commands/project_base.rb:19:in `execute'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/exec/sub_command_ui.rb:43:in `perform!'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/lib/compass/exec/sub_command_ui.rb:15:in `run!'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/bin/compass:30:in `block in <top (required)>'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/bin/compass:44:in `call'
  C:/Ruby193/lib/ruby/gems/1.9.1/gems/compass-0.12.6/bin/compass:44:in `<top (required)>'
  C:/Ruby193/bin/compass:23:in `load'
  C:/Ruby193/bin/compass:23:in `<main>'
```

Here is my `config.rb` file, in its entirety (reduced to generate this test case):

```
require "compass/import-once/activate"
```

My reduced test case requires two stylesheets, named `sass\Admin.scss` and `sass\Public.scss`.  They both have this identical content:

```
@import "small/*.png";
```

My reduced test case requires some suitable images in the `images\small` folder.  Any images will do.

Removing any line from this reduced test case makes the problem go away.

**Comment by chriseppstein** [comment]
I cannot reproduce this. Please retest with the 1.0 release candidate and let me know if it's still happening. If so, I guess I need this test case as a git repo or zip file.

**Comment by chriseppstein** [comment]
I was able to reproduce it afterall.

**Comment by chriseppstein** [comment]
Fixed in `compass-import-once` version `1.0.5`

**Comment by PabloVallejo** [comment]
Working with `1.0.5` for me as well :+1:
